docker-app/qfieldcloud/core/models.py

```diff
@@ -1159,6 +1098,28 @@ def clean(self) -> None:
         return super().clean()
 
 
+class ProjectPermissionsView(models.Model):
+    user = models.ForeignKey(
+        User,
+        on_delete=models.DO_NOTHING,
+        related_name="related_projects",
+    )
+    project = models.OneToOneField(
+        "Project",
+        on_delete=models.DO_NOTHING,
+        related_name="user_role",
```

Comment:
Here too maybe we could come up with a more clear related name (either change the model name to `ProjectRoles`, or also use `projects_permissions` as related name and then maybe rename the `name` field to `role_name` ?

Reply:
I would go for `ProjectRolesView` in such case. We have more `roles` than permissions here.

Reply:
yes sounds good